packages/@angular/cli/commands/completion.ts

```diff
@@ -117,14 +117,15 @@ const CompletionCommand = Command.extend({
         });
       }
 
-      let opts = '';
+      let opts: string[] = [];
       if (command.blueprints && command.blueprints[0]) {
-        opts += extractBlueprints(command.blueprints);
+        opts = opts.concat(extractBlueprints(command.blueprints));
       }
 
       if (command.availableOptions && command.availableOptions[0]) {
```

Comment:
There are no tests regarding the output of copmletion.
I just saw the issue with the missing space when testing angular-cli and tracked it down.